src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -352,6 +352,336 @@
   </div>
 </div>
 
+{% if responseType equals "FULL_ADMISSION" %}
+{% if paymentOption is defined %}
+<div class="section">
+  <div class="section-heading">
+    <strong>When the defendant will pay</strong>
+  </div>
+    <div class="section-body">
+      <table>
+        <tr>
+          <td><strong>When will they pay:</strong></td>
+          <td>
+            {{ paymentOption }}
+          </td>
+        </tr>
+
+        {% if repaymentPlan is defined and repaymentPlan.repaymentOption == "By instalments" %}
+        <tr class="no-border">
+          <td><strong>Regular payments of:</strong></td>
+          <td>
+            {{ repaymentPlan.instalmentAmount }}
+          </td>
+        </tr>
+        <tr class="no-border">
+          <td><strong>Frequency of payments:</strong></td>
+          <td>
+            {{ repaymentPlan.paymentSchedule }}
+          </td>
+        </tr>
+        <tr class="no-border">
+          <td><strong>Make the first payment on:</strong></td>
```

Comment:
No need to show payment deadline for payment by set date? Can you double check with business / UX / UC?

Reply:
Its not deadline, it first payment date and only displayed for `repaymentPlan.repaymentOption == "By instalments"`

Reply:
I mean there is also `paymentDate` property that is populated for immediate payment and payment in full by set date. I am asking why we are not showing that on the PDF.

Reply:
added that in whenWillTheyPay